src/pages/home-page.js

```diff
@@ -0,0 +1,186 @@
+import React from 'react'
+import './home.css'
+
+export default function HomePage() {
+  return (
+    <div className="container-fluid">
+      {/* Logo */}
+      <h1 className="text-center mb-5" id="logo-page">
+        Pokemon Dex
+      </h1>
+      {/* ! Search Form*/}
+      <form id="form-search" className="mb-5">
+        <label htmlFor="SearchPokemon" className="form-label">
+          Search pokemon
+        </label>
+        <input
+          type="text"
+          className="form-control"
+          id="SearchPokemon"
```

Comment:
`id` should be in hyphen-separated case `search-pokemon-input`

Reply:
fixed Issue